multicluster/cmd/multicluster-controller/gateway_webhook.go

```diff
@@ -47,13 +49,21 @@ func (v *gatewayValidator) Handle(ctx context.Context, req admission.Request) ad
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	ui := req.UserInfo
+	_, saName, err := serviceaccount.SplitUsername(ui.Username)
+	if err != nil {
+		klog.ErrorS(err, "Error getting ServiceAccount name", "Gateway", req.Namespace+"/"+req.Name)
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+	if saName != mcControllerSAName {
```

Comment:
We should at least allow other users to get Gateway right?

Reply:
We defined a Gateway webhook to validate the operation for CREATE/UPDATE, it has no impact to `GET`.

Reply:
An operation check is added above the SA check. Now only "antrea-mc-controller" could update or create Gateways, but other users could also get Gateways. Two more unit test cases were added.